app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/Hendelse.kt

```diff
@@ -35,18 +35,55 @@ sealed class Hendelse {
         val eksternId: String,
         val mottaker: Mottaker,
         val tekst: String,
-        val grupperingsid: String? = null,
+        val grupperingsid: String?,
         val lenke: String,
         val opprettetTidspunkt: OffsetDateTime,
+        val eksterneVarsler: List<EksterntVarsel>,
+    ) : Hendelse() {
 
-        // TODO: Det finnes hendelser i kafka-strømmen som mangler dette feltet.
-        // Men det er dumt at vi må sette default value for hele prosjektet, siden
-        // det kan være man glemmer å sette den, og på den måten får bugs.
-        val eksterneVarsler: List<EksterntVarsel> = listOf()
-    ) : Hendelse()
+        companion object {
+            // Denne konstruktøren har default properties, og støtter historiske
+            // JSON-felter man kan finne i kafka-topicen.
+            // Denne konstruktøren skal ikke brukes i vår kode, fordi da er det lett å gå glipp
+            // av å initialisere viktige felt.
+            @JsonCreator
+            @JvmStatic
+            fun jsonConstructor(
+                virksomhetsnummer: String,
+                notifikasjonId: UUID,
+                hendelseId: UUID,
+                produsentId: String,
+                kildeAppNavn: String,
+                merkelapp: String,
+                eksternId: String,
+                mottaker: Mottaker? = null,
+                mottakere: List<Mottaker> = listOf(),
+                tekst: String,
+                grupperingsid: String? = null,
+                lenke: String,
+                opprettetTidspunkt: OffsetDateTime,
+                eksterneVarsler: List<EksterntVarsel> = listOf(),
+            ) = BeskjedOpprettet(
+                virksomhetsnummer = virksomhetsnummer,
+                notifikasjonId = notifikasjonId,
+                hendelseId = hendelseId,
+                produsentId = produsentId,
+                kildeAppNavn = kildeAppNavn,
+                merkelapp = merkelapp,
+                eksternId = eksternId,
+                mottaker = mottaker ?: mottakere[0],
```

Comment:
hvis vi får mer enn en mottaker burde vi kanskje feile hardt slik at vi stopper prosessering.
Det er jo ikke noe som kan skje slik det er nå, men hvis vi glemmer å implementere støtte for flere mottakere det før vi oppretter hendelser med flere mottakere så kan vi havne i en bad state.